Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -36,6 +36,9 @@
     '2022HIRP', #RawPrime
     '2023HI',
     '2023HIRP', #RawPrime
+    '2024HLTOnDigi',
+    '2024HLTOnDigiPU'
+
```

Comment:
Do we need space here? Maybe it can be fixed later when new workflow comes.

Reply:
Not really, right. Let me fix it in one of the other PRs I have or will open on upgrade wfs.